crates/astria-composer/tests/blackbox/helper/mock_sequencer.rs

```diff
@@ -57,3 +59,62 @@ pub async fn mount_abci_query_mock(
         .mount_as_scoped(server)
         .await
 }
+
+async fn mount_cometbft_status_response(
+    server: &MockServer,
+    mock_sequencer_chain_id: &str,
+) -> MockGuard {
```

Comment:
No need to return the mock guard into the builder. There isn't actually something to assert on other that the mock guard was triggered. Just mount it non-scoped and don't return a guard.